modules/500-upmeter/docs/internal/DEVELOPMENT_RU.md

```diff
@@ -2,29 +2,89 @@
 title: "Информация для разработчиков"
```

Comment:
I'd call it CONCEPTS, not 'development'.

Reply:
OK. Made the changes.